sdk/maps/Azure.Maps.Routing/src/Models/Options/RouteDirectionOptions.cs

```diff
@@ -219,30 +219,30 @@ public RouteDirectionOptions()
         /// Sensible Values : 50,8.2:130,21.3
         /// This parameter is required for <c>Electric consumption model</c>.
         /// </summary>
-        public string ConstantSpeedConsumptionInKwHPerHundredKm { get; set; }
+        public string ConstantSpeedConsumptionInKilowattHourPerHundredKilometer { get; set; }
 
         /// <summary>
         /// Specifies the current electric energy supply in kilowatt hours (kWh).
-        /// This parameter co-exists with <c>maxChargeInkWh</c> parameter.
-        /// The range of values allowed are 0.0 to <c>maxChargeInkWh</c>.
+        /// This parameter co-exists with <c>MaxChargeInKilowattHour</c> parameter.
+        /// The range of values allowed are 0.0 to <c>MaxChargeInKilowattHour</c>.
         /// Sensible Values : 43
         /// </summary>
-        public double? CurrentChargeInKwH { get; set; }
+        public double? CurrentChargeInKilowattHour { get; set; }
 
         /// <summary>
         /// Specifies the maximum electric energy supply in kilowatt hours (kWh) that may be stored in the vehicle's battery.
-        /// This parameter co-exists with <c>currentChargeInkWh</c> parameter.
-        /// Minimum value has to be greater than or equal to <c>currentChargeInkWh</c>.
+        /// This parameter co-exists with <c>currentChargeInkilowattHour</c> parameter.
+        /// Minimum value has to be greater than or equal to <c>currentChargeInkilowattHour</c>.
         /// Sensible Values : 85
         /// </summary>
-        public double? MaxChargeInKwH { get; set; }
+        public double? MaxChargeInKilowattHour { get; set; }
```

Comment:
Shouldn't these all be pluralized?

`MaxChargeInKilowattHours`,  `AuxiliaryPowerInKilowatts` and so on?